module1/src/test/java/net/broscorp/primitives/PrimitivesTest.java

```diff
@@ -0,0 +1,50 @@
+package net.broscorp.primitives;
+
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+public class PrimitivesTest {
+
+  @Test
+  public void overflowTest() {
+
+    //variable is in positive range
+    int testedVariable = 1;
+    //adding positive number
+    testedVariable += Integer.MAX_VALUE;
+
+    //for some reason variable is now negative(((
+    assertTrue(testedVariable < 0);
+
+  }
```

Comment:
Add test to illustrate underflow

Reply:
i guess that's overcoding, underflow is basically the same mechanism as overflow